spec/controllers/hq/partners_controller_spec.rb

```diff
@@ -20,26 +20,52 @@ def foobar_action
 end
 
 RSpec.describe Hq::PartnersController, type: :controller do
-  before :each do
-    sign_in instance_double(AdminUser)
-    @partner = build_stubbed :partner
-    expect(Partner).to receive(:find).and_return(@partner)
-  end
 
-  specify 'editing renders the edit view' do
-    get :edit, id: @partner.id
-    expect(response).to render_template 'edit'
-  end
+  context '#new and #create' do
+    
+    before :each do
+      sign_in instance_double(AdminUser)
+      @partner = FactoryGirl.build(:partner)
+    end
+    
+    specify 'successful create redirects to the show view' do
+      post :create, partner: @partner.attributes
+      @partner = Partner.find_by_name(@partner[:name])
+      expect(response).to redirect_to hq_partner_path(@partner)
+    end
```

Comment:
Have you tried using an `instance_double` or a simple `double` instead? Something along the lines of
```ruby
specify 'successful create redirects to the show view' do
  @partner = instance_double Partner, :id => 1, :save => true

  post :create, partner: {}

  expect(response).to redirect_to hq_partner_path(1)
end
```